dpdata/plugins/deepmd.py

```diff
@@ -69,6 +71,92 @@ def from_labeled_system(self, file_name, type_map=None, **kwargs):
     MultiMode = Format.MultiModes.Directory
 
 
+@Format.register("deepmd/npy/mixed")
+class DeePMDMixedFormat(Format):
```

Comment:
Could you add documentation here for this format?
(I am not sure what is the best way to do it, but we should have an automatic way to have documentation for so many different formats)